runelite-client/src/main/java/net/runelite/client/plugins/slayer/Task.java

```diff
@@ -123,7 +123,7 @@
 	KURASK("Kurask", ItemID.KURASK),
 	ROGUES("Rogues", ItemID.ROGUE_MASK, "Rogue"),
 	LESSER_DEMONS("Lesser demons", ItemID.LESSER_DEMON_MASK),
-	LIZARDS("Lizards", ItemID.DESERT_LIZARD, "Desert lizard", "Sulphur lizard", "Small lizard", "Lizard"),
+	LIZARDS("Lizards", ItemID.DESERT_LIZARD, 4, ItemID.ICE_COOLER, "Desert lizard", "Small lizard", "Lizard"),
```

Comment:
sulphur lizards are a valid lizard task, why are you removing them

Reply:
I added the ItemID.ICE_COOLER because lizards requires that item to be kill, not the Sulphur lizard.  I noticed: `Players can kill these as an alternative to Desert Lizards, provided they have the higher Slayer level. However, sulphur lizards do not require ice coolers to finish off. `

We could keep the sulphur lizard in the tast, but add an ice cooler.

Reply:
from discussion in discord, just adding back the sulphur lizard would undesirably show the weakness overlay on them when it doesnt need to be.  A suggestion was proposed that another list of targets that dont have a weakness could be added, something like `Task(String name, int itemSpriteId, String[] targetNames, int weaknessThreshold, int weaknessItem, String... affectedTargetNames)`,  and just use that to tell the overlay to only draw the weakness on the appropriate monsters.